Mathlib/Algebra/Group/Conj.lean

```diff
@@ -114,7 +114,7 @@ theorem conj_zpow {i : ℤ} {a b : α} : (a * b * a⁻¹) ^ i = a * b ^ i * a⁻
   induction' i
   · change (a * b * a⁻¹) ^ (_ : ℤ) = a * b ^ (_ : ℤ) * a⁻¹
     simp [zpow_coe_nat]
-  · simp [zpow_negSucc, conj_pow]
+  · simp only [zpow_negSucc, conj_pow, mul_inv_rev, inv_inv]
     rw [mul_assoc]
```

Comment:
Make the proof more robust by not squeezing the simp:
```suggestion
  · simp [zpow_negSucc, conj_pow, mul_assoc]
```

Reply:
Done.